touchforms/backend/xformplayer.py

```diff
@@ -230,6 +283,27 @@ def ix_in_scope(form_ix):
             else:
                 return FormIndex.isSubElement(parent_ix, form_ix)
 
+
+        def print_node(node, acc, a):
+
+            attribute_count = node.getAttributeCount()
+            for x in range (0, attribute_count):
```

Comment:
`xrange` is typically better for this because it does not actually allocate an entire list in memory